### PR TITLE
Roland DJ-505: Add saved loop mode

### PR DESCRIPTION
**Pull request by Holzhaus**
Depends on #2194.

**EDIT:** Documentation updates can be found in https://github.com/mixxxdj/manual/pull/243

### CODE REVIEW

**Comment by Holzhaus** [comment]
Ok, ready to review. Please also review and merge the updated documentation at: https://github.com/mixxxdj/manual/pull/243

**Comment by Holzhaus** [comment]
Appveyor failures are unrelated.

**Comment by Holzhaus** [comment]
Merge?

File: res/controllers/Roland_DJ-505-scripts.js [hunk 1]
```diff
@@ -496,8 +496,8 @@ DJ505.Deck = function(deckNumbers, offset) {
 
     this.loopActive = new components.Button({
         midi: [0x94 + offset, 0x32],
-        inKey: "reloop_toggle",
-        outKey: "loop_enabled",
+        key: "loop_enabled",
```
Comment (Be-ing) [inline res/controllers/Roland_DJ-505-scripts.js:499]: `loop_enabled` is read/write now?

**Pull request merged**